plans/tier0.fmf

```diff
@@ -0,0 +1,43 @@
+discover+:
+    tier: 0
+
+/changed_yum_conf:
+  discover+:
+    test: changed-yum-conf
+
+/check_custom_repo:
+  discover+:
+    test: check-custom-repo
+
+/check_variant_message:
+  discover+:
+    test: check-variant-message
+
+/inhibit_if_kmods_is_not_supported:
+  discover+:
+    test: inhibit-if-kmods-is-not-supported
+
+/inhibit_if_oracle_system_uses_not_standard_kernel:
```

Comment:
Now this plans is only for Centos8, we should not run this one as it's oracle specific